src/tls_wrap.cc

```diff
@@ -90,8 +90,10 @@ TLSWrap::~TLSWrap() {
   MakePending();
 
   // And destroy
-  while (WriteItem* wi = pending_write_items_.PopFront())
+  while (WriteItem* wi = pending_write_items_.PopFront()) {
+    wi->w_->Done(UV_ECANCELED);
```

Comment:
Because `WriteWrap`s are not destroyed in `stream_base.cc` (and previously in `stream_wrap.cc`) unless the callback is invoked.